bbcode-src/tags/font.js

```diff
@@ -1,12 +1,4 @@
-// import { getUniqAttr } from "@bbob/plugin-helper";
-
 export const font = (node, options) => {
-  // console.log(node);
-  // console.log(node.attrs);
-  // console.log(getUniqAttr(node.attrs));
-  // options.data = [...options.data, 1];
-  // eslint-disable-next-line no-console
-  console.log(options);
```

Comment:
I am sure this was here to avoid the linting error that comes from options not being used. Once we fully implement fonts, this will no longer be an issue.